package.json

```diff
@@ -19,17 +19,44 @@
     "test": "apm test"
   },
   "configSchema": {
-    "markerType": {
+    "markerTypeError": {
       "type": "string",
       "default": "line",
       "enum": [
+        "off",
         "line",
         "gutter",
         "highlight-under",
         "highlight-over",
         "highlight-outline"
       ],
-      "description": "Marker type for linter highlights"
+      "description": "Marker type for `severity:Error` highlights"
```

Comment:
Change this to just "for error highlights", users aren't going to know what `severity:Error` means 😉.